### DISCUSSION TITLE
empty messageBox - add Share Account

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#6600] [Q&A] [answered]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/b55119b9-10a8-4bbb-82fa-518d35bc579f)

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/9ad7e96e-cae6-4fb2-b851-82bab800b110)

When I register share account, a message box appears, and the contents are empty.
I have completed all related translations.

why empty ? ? ?

**Comment by n-stein** [comment, marked as answer]
It's not blank for me, but also doesn't appear translated:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/f7e13f8d-73a2-4c09-96bc-5a8409f3fead)

There is an issue with the translations of strings which are retrieved via wxString::FromUTF8, mostly those with the `→` character such as this one. I have created #6614 for the fix.
